src/libusb/handle-libusb.h

```diff
@@ -0,0 +1,57 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "types.h"
+#include <chrono>
+
+#include <libusb.h>
+
+namespace librealsense
+{
+    namespace platform
+    {
+        class handle_libusb
+        {
+        public:
+            handle_libusb(libusb_device* device, uint8_t interface, int timeout_ms = 100) :
```

Comment:
Add a comment how this value was selected

Reply:
timeout will be removed from the handle signature